corehq/motech/repeaters/migrations/0015_drop_receiverwrapper_couchdb.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.2.25 on 2024-04-02 14:38
+
+from django.conf import settings
+from django.db import migrations
+
+from corehq.util.django_migrations import skip_on_fresh_install
+from couchdbkit.client import Database, ResourceNotFound
+
+
+@skip_on_fresh_install
+def _delete_receiverwrapper_couchdb():
+    db = Database(f"{settings.COUCH_DATABASE}__receiverwrapper")
+    try:
+        db.server.delete_db(db.dbname)
+    except ResourceNotFound:
+        pass
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('repeaters', '0011_remove_obsolete_entities'),
```

Comment:
Is it intentional that this doesn't depend on the most recent repeaters migration, `0014_alter_repeater_request_method`?

Reply:
No. That was an oversight. Fixing.

Reply:
https://github.com/dimagi/commcare-hq/pull/35278/commits/1968d57843b23e16e4fcd7ab7b76b9e893cad03a